CHANGES

```diff
@@ -9,19 +24,22 @@ Version 0.4.0 (released 2015-03-11)
   provided. (#50)
 - PEP8 improvements.
 
-Version 0.3.0 (released 2014-11-05):
+Version 0.3.0 (released 2014-11-05)
+-----------------------------------
 
```

Comment:
These changes mess up the table of contents a bit, see below.  Let's clean it or let's get back to the old style (visible on http://dictdiffer.rtfd.org/)?

![2016-01-04-150835_2560x1440_scrot](https://cloud.githubusercontent.com/assets/517546/12091173/56a9b902-b2f5-11e5-8431-26b287dac125.png)